src/docfx/Errors.cs

```diff
@@ -300,13 +300,13 @@ public static Error InvalidMetadataType(SourceInfo source, string name)
         /// Failed to compute specific info of a commit.
         /// </summary>
         public static Error GitLogError(string repoPath, int errorCode)
```

Comment:
@mingwli, I think we can remove this error code and make it an `NotSupportedException`, if we reach here, better let docfx crash